### PR TITLE
Radau points

### PR DESCRIPTION
**Pull request by rckirby**
This does a few things:
1.) It provides the Radau quadrature rule, a classical family akin to Gauss/Jacobi/Lobatto/etc methods heretofore missing from FIAT.
2.) It gives a discontinuous finite element based on the Radau points.

Although I doubt users will directly apply these elements to solve PDE, I am using FIAT in another project to generate arbitrary-order Butcher tableaux for collocation-type Runge-Kutta methods, and these are exactly what I need to generate the classical RadauIIA family.

### CODE REVIEW

File: FIAT/gauss_radau.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2015 Imperial College London and others.
```
Comment (miklos1) [inline FIAT/gauss_radau.py:1]: Copy-paste copyright notice

**Comment by garth-wells** [comment]
Some tests would be nice.

**Comment by rckirby** [comment]
Added on the radau branch.

**Comment by rckirby** [comment]
Tests added, everything is passing on CircleCI.  Are y'all ready to merge?

**Comment by rckirby** [comment]
Nudging this so we can hopefully merge and move on.
Thanks!

**Comment by rckirby** [comment]
Thanks!

**Pull request merged**